specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -3131,6 +3131,9 @@
           },
           "Create a VM with UserData": {
             "$ref": "./examples/CreateOrUpdateAVmWithUserData.json"
+          },
+          "Create a VM with Capacity Reservation": {
+            "$ref": "./examples/CreateOrUpdateAVmWithCapacityReservation.json"
```

Comment:
should it be createorupdate? or just create? i see most of the entries here are just create

Reply:
updated